changelogs/fragments/private_really.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - ensure private attributes stay private.
```

Comment:
looking to files modified I assume this is only for playbook (I did not know playbook had this), if this is the case, that could be added in the sentence.

Reply:
yep, only playbook objects have attributes .. but only very few devs actually know that, i'll update